launch/image_processing.launch

```diff
@@ -0,0 +1,15 @@
+<launch>
+  <arg name="view" default="false" /> 
+
+
+  <node name="process_node" pkg="robosub" type="process_node.py">
+  	<param name="target" value="image_target"/>
```

Comment:
Looks like the indentation is a bit inconsistent. You indented the level above with 2 spaces, then used 4 here